Доработка приложения согласно заданию 8-го спринта

texport opened the pull request.

Commit:
Доработка приложения согласно заданию 8-го спринта

amardanov commented on ImageFeed/ImagesList/ImagesListCell.swift:
```diff
@@ -0,0 +1,90 @@
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        self.backgroundColor = UIColor(named: "YP Black")
```
Лучше перенести этот код в init, так как задание цвета нам нужно только 1 раз, а `layoutSubviews` вызывается несколько раз за время жизни ячейки

amardanov commented on ImageFeed/ImagesList/ImagesListCell.swift:
```diff
@@ -0,0 +1,90 @@
+    
+    private let container = UIView()
+    
+    let customImageView: UIImageView = {
```
Давай сделаем все эти вьюшки приватными, а их конфигурацию вынесем в самописную функцию configure(\*перечисление аргументов*\). Таким образом мы скрываем детали реализации UI в ячейке и убираем доступ к конфигурации извне

Commit:
- self.backgroundColor = UIColor(named: "YP Black") перенес в init
- configure(*перечисление аргументов*) добавил новый метод и убрал вью в приват

Commit:
- Добавил градиент к customLabel
